sdks/java/core/src/main/java/org/apache/beam/sdk/util/ApiSurface.java

```diff
@@ -86,7 +86,7 @@
  * not interesting.
  */
 @SuppressWarnings("rawtypes")
-public class ApiSurface {
+public abstract  class ApiSurface {
```

Comment:
I see what you are saying now ApiSurface is not meant to be defined by modules just to verify the set of entities they expose. will make the change accordingly. Thanks for the review.